src/styles/blocks/about.scss

```diff
@@ -0,0 +1,43 @@
+.about {
```

Comment:
please add background color as in the Figma layout (we shouldn't see edges of images with bikes):
![image](https://user-images.githubusercontent.com/86923089/191076890-762fe6e9-7032-48f8-8fa7-2c5f220092c1.png)